policy/modules/kernel/filesystem.if

```diff
@@ -6136,6 +6136,43 @@ interface(`fs_getattr_tracefs_files',`
         allow $1 tracefs_t:file getattr;
 ')
 
+########################################
+## <summary>
+##	Read/write trace filesystem files
+## </summary>
+## <param name="domain">
+##      <summary>
+##      Domain allowed access.
+##      </summary>
+## </param>
+#
+interface(`fs_write_tracefs_files',`
+	gen_require(`
+		type tracefs_t;
+	')
+
+	allow $1 tracefs_t:dir list_dir_perms;
+	allow $1 tracefs_t:file rw_file_perms;
+')
```

Comment:
A bit of inconsistency. Since everything but the interface name is `rw`, I assume the interface name should be `rw` instead.